Exiled.API/Features/Scp330.cs

```diff
@@ -0,0 +1,162 @@
+// -----------------------------------------------------------------------
```

Comment:
1. Item features belong in the Features/Items/ sub-directory
2. This item already exists in the API in Features/Items/Scp330.cs